src/sidebars/search/search-controller.js

```diff
@@ -9,6 +9,9 @@
             this.selectionService = selectionService;
             this.currentWindowService = currentWindowService;
 
+            this.quandlService.on('CONNECTION_STATUS_CHAGED', this._setConnectionStatus, this);
```

Comment:
You might be able to replace `_setConnectionStatus` with something like, `(status) => { this.isConnected = status; }`, that way you don't need to pass references to `this` around